staker/tests/__TEST_staker_short_warmup_period_external_12_test.gnoA

```diff
@@ -157,6 +157,6 @@ func TestCollectReward(t *testing.T) {
 	newBar := bar.BalanceOf(a2u(gsa))
 	newQux := qux.BalanceOf(a2u(gsa))
 
-	shouldEQ(t, newBar > oldBar, true)
-	shouldEQ(t, newQux > oldQux, true)
+	shouldEQ(t, bool(newBar > oldBar), true)
+	shouldEQ(t, bool(newQux > oldQux), true)
```

Comment:
(Just for memo) need to investigate what the type is inferred when comparing values.